src/audioWorklet/amplitudeProcessor.js

```diff
@@ -0,0 +1,93 @@
+// import processor name via preval.require so that it's available as a value at compile time
+const processorNames = preval.require('./processorNames');
+
+class AmplitudeProcessor extends AudioWorkletProcessor {
+  static get parameterDescriptors() {
+    return [
+      {
+        name: 'smoothing',
+        defaultValue: 0,
+        minValue: 0,
+        maxValue: 1,
+        automationRate: 'k-rate'
```

Comment:
inspecting this in Chrome 75.0.3770.100, somehow it has become `'a-rate'`. Confusing, but looks like you're following the spec https://webaudio.github.io/web-audio-api/#ref-for-enumdef-automationrate%E2%91%A1

Reply:
yeah, I see the bug, it looks like the polyfill is assuming that all worklet parameters are `a-rate`: https://github.com/GoogleChromeLabs/audioworklet-polyfill/blob/d6f1a14dacc597876b1a5f9cd8719729a131b585/src/index.js#L96

I think your suggested approach makes sense - another possible solution is to treat the smoothing parameter as `a-rate` (i.e. a Float32 buffer of values) and update the processor code accordingly. This seems more in line with the intended use case of AudioParams, although probably less efficient for non-AudioWorklet browsers. What's your opinion?